### ISSUE TITLE
Feature Request: Add an option next to each day to state that you didn't get the prices for that day (closed, missed etc)

### ISSUE THREAD

**Issue by JamieBohannaWebDev** [mikebryant/ac-nh-turnip-prices#423]
**Is your feature request related to a problem? Please describe.**
There is no method of entering in 0 or skipping a day's prices.

**Describe the solution you'd like**
We should add some method of a tickbox next to each time slot to state that the price was unavailable (Nook's Cranny under development or closed, or you simply missed/forget to get that time's turnip prices).

**Describe alternatives you've considered**
I tried entering in 0, however this bugged the chart out and set the min/max to infinity.

**Comment by theRTC204** [comment]
You can just leave the field empty.

**Comment by JamieBohannaWebDev** [comment]
@theRTC204 Of course, I was just referring to adding this which allows a more refined/improved user experience

**Comment by jakobsieger** [comment]
I can see @theRTC204's point here that you don't actually need it. But in my opinion this is indeed an improvement in user experience, as @JamieBohannaWebDev already meantioned. It's a good idea!
